### PR TITLE
Replace gopkg.in/auth0.v5 with github.com/auth0/go-auth0

### PR DESCRIPTION
**Pull request by sergiughf**
### Description

This PR replaces gopkg.in/auth0.v5 (github.com/go-auth0/auth0) with github.com/auth0/go-auth0 as we now have brought the SDK within the Auth0 org. 

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used, if not `master`

### CODE REVIEW

File: internal/auth0/action.go [hunk 1]
```diff
@@ -28,10 +28,10 @@ type ActionAPI interface {
 	// See: https://auth0.com/docs/api/management/v2#!/Actions/get_actions
 	List(opts ...management.RequestOption) (c *management.ActionList, err error)
 
-	// ListTriggers available.
+	// Triggers available.
 	//
 	// https://auth0.com/docs/api/management/v2/#!/Actions/get_triggers
-	ListTriggers(opts ...management.RequestOption) (l *management.ActionTriggerList, err error)
+	Triggers(opts ...management.RequestOption) (l *management.ActionTriggerList, err error)
```
Comment (sergiughf) [inline internal/auth0/action.go:34]: The `ListTriggers` has been deprecated in favor of `Triggers`. The func signature remains the same.

File: internal/cli/actions.go [hunk 2]
```diff
@@ -201,7 +202,7 @@ auth0 actions create --n myaction -t post-login -d "lodash=4.0.0" -s "API_KEY=va
 
 			action := &management.Action{
 				Name: &inputs.Name,
-				SupportedTriggers: []management.ActionTrigger{
+				SupportedTriggers: []*management.ActionTrigger{
```
Comment (sergiughf) [inline internal/cli/actions.go:205]: This has also been changed to a slice of pointers within the Go SDK.

File: internal/cli/organizations.go [hunk 3]
```diff
@@ -212,11 +213,11 @@ auth0 orgs create --n myorganization -d "My Organization" -m "KEY=value" -m "OTH
 				o.Branding = &management.OrganizationBranding{}
 
 				if isLogoURLSet {
-					o.Branding.LogoUrl = &inputs.LogoURL
+					o.Branding.LogoURL = &inputs.LogoURL
 				}
 
 				if isAnyColorSet {
-					o.Branding.Colors = map[string]string{}
+					o.Branding.Colors = map[string]interface{}{}
```
Comment (sergiughf) [inline internal/cli/organizations.go:220]: This has also changed within the Go SDK, however the underlying field values remain strings. That's why further down we cast again to `.(string)`.

**Review by sergiughf — COMMENTED** [review]
Adding comments around the code to give further context on some breaking changes withing the Go SDK. 

PRs that introduced the breaking changes:
- https://github.com/go-auth0/auth0/pull/247
- https://github.com/go-auth0/auth0/pull/246

**Review by willvedd — APPROVED** [review]

**Pull request merged**